fixtures/gists-app/app/components/Counter.js

```diff
@@ -1,9 +1,14 @@
 import { useState } from "react";
+import styles from "./Counter.module.css";
```

Comment:
Is this decomposable? Could I do `import { button } from "./Counter.module.css"`

Reply:
Yes. All I'm doing is resolving this import as a JSON module that maps the object key to the hashed classname, so it'll work just like any other object import.